Sources/Amplitude/Amplitude.m

```diff
@@ -1677,8 +1685,50 @@ - (id)unarchive:(NSString*)path {
     return nil;
 }
 
+- (id)unarchive:(NSData *)data error:(NSError **)error {
```

Comment:
Do you feel that this function should be better moved to `AMPUtils`?

Reply:
I feel honoured to be asked for my opinion to this, not being a member of your organisation :)

I thought about this while implementing the method, but since `- (BOOL)archive:(id) obj toFile:(NSString*)path;` and `- (id)unarchive:(NSString*)path`, which is the only caller of my `unarchive` method are implemented in Amplitude.m as well, I implemented the unarchive function there as well, to try and keep things in the same place.

Also, as far as I see it, both `- (BOOL)archive:(id) obj toFile:(NSString*)path;` and`- (id)unarchive:(NSString*)path` are only used inside Amplitude.m (and the Tests category) and are not added to the corresponding header file (Amplitude.h), which imho would suggest leaving them there.

Reply:
We're always happy to see developers contributing to our project! And hope you enjoy using Amplitude product too.

Sure, that make sense to me.